external/dhcpc/dhcpc.c

```diff
@@ -118,6 +118,9 @@
 
 #define BUFFER_SIZE             256
 
+
+#define MAX_DNS                 2
```

Comment:
Could you follow the macro definition with a comment to describe what the macro is?

Reply:
#define MAX_DNS                        2   //<description>